handbook/product-design/product-design.rituals.yml

```diff
@@ -23,7 +23,7 @@
   task: "Sprint kickoff review" # 2024-03-06 TODO: Link to responsibility or corresponding "how to" info e.g. https://fleetdm.com/handbook/company/product-groups#making-changes
   startedOn: "2024-03-07" 
   frequency: "Triweekly" 
-  description: "Review stories that made it into this sprint and stories that didn't make it into this sprint. Ensure stories/bugs have been effectively prioritized across teams. For stories that didn't make it into the sprint, API design DRI updates the reference docs release branches." 
```

Comment:
FYI @rachaelshaw also removed this bit now that it's covered in the "Review reference docs for upcoming release" ritual.